tests/__init__.py

```diff
@@ -3,7 +3,11 @@
 import os
 from uuid import uuid4
 
-from tests.fixtures.rsa_pair import sample_private_key, sample_public_key
+from tests.fixtures.rsa_pair import (  # noqa:F401
```

Comment:
what this `noqa` is for here?

Reply:
unused import

Reply:
So, I think we shouldn't import them here at all 😉

Reply:
nope, we should - this simplifies imports.

Reply:
That is true but all _"fixtures"_ should be gathered in `tests/fixtures/__init__.py`, not here 😉